gauge-display/script.js

```diff
@@ -248,63 +245,40 @@ function chartNight(rimFill, faceColor) {
 
         gaugeRim.setAttribute('fill', rimFill);
         gaugeBackground.setAttribute('fill', faceColor);
-
     })
 }
 
 let longTouchTimer;
-var touchduration = 500;
+const TOUCH_DURATION = 500;
 const gMeterDiv = document.getElementById('gmeter-main');
-gMeterDiv.addEventListener('touchstart', () => longTouchTimer = setTimeout(resetGmeter, touchduration))
+gMeterDiv.addEventListener('touchstart', () => longTouchTimer = setTimeout(resetGMeter, TOUCH_DURATION))
 gMeterDiv.addEventListener('touchend', () => clearTimeout(longTouchTimer));
 
-let resetGmeter = function () {
+function resetGMeter() {
     localStorage.setItem('xOffset', currentAccelX);
     localStorage.setItem('yOffset', currentAccelY);
     accelXOffset = currentAccelX;
     accelYOffset = currentAccelY;
 }
 
 function changeLedBrightness(isUp) {
-    if (isUp && night) {
-        ledBrightnessNight++
-        if (ledBrightnessNight > 255) {
-            ledBrightnessNight = 255;
-        }
+    const clampBrightness = (value) => Math.min(Math.max(value, 0), 255);
+    if (night) {
+        isUp ? ledBrightnessNight++ : ledBrightnessNight--;
+        ledBrightnessNight = clampBrightness(ledBrightnessNight);
         localStorage.setItem('brightness-night', ledBrightnessNight);
-        fetch('/change-night?brightness=' + ledBrightnessNight)
-            .then(data => console.log(data))
-    } else if (isUp && !night) {
-        ledBrightnessDay = ledBrightnessDay + 20
-        if (ledBrightnessDay > 255) {
-            ledBrightnessDay = 255;
-        }
-        localStorage.setItem('brightness-day', ledBrightnessDay);
-        fetch('/change-night?brightness=' + ledBrightnessDay)
-            .then(data => console.log(data))
-    }
-
-    if (!isUp && night) {
-        ledBrightnessNight--;
-        if (ledBrightnessNight < 0) {
-            ledBrightnessNight = 0;
-        }
-        localStorage.setItem('brightness-night', ledBrightnessNight);
-        fetch('/change-night?brightness=' + ledBrightnessNight)
-            .then(data => console.log(data))
-    } else if (!isUp && !night) {
-        ledBrightnessDay = ledBrightnessDay - 20;
-        if (ledBrightnessDay < 0) {
-            ledBrightnessDay = 0;
-        }
+    } else {
+        isUp ? ledBrightnessDay += 20 : ledBrightnessDay -+ 20;
+        ledBrightnessDay = clampBrightness(ledBrightnessDay);
         localStorage.setItem('brightness-day', ledBrightnessDay);
-        fetch('/change-night?brightness=' + ledBrightnessDay)
-            .then(data => console.log(data))
     }
+    const brightnessToSet = night ? ledBrightnessNight : ledBrightnessDay;
```

Comment:
Rather than changing the increment/decrement step values you could use a daytime multiplier.
```suggestion
    const brightnessToSet = night ? ledBrightness : ledBrightness * 20;
```